images/iqsharp-base/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # The same image used by mybinder.org
-FROM python:3.7-slim
+FROM python:3.7-slim-stretch
```

Comment:
do we want to a dynamic label, or should we attach to a specific version?

Reply:
From my understanding, this is the almost the most specific tag offered by the Python image repo. We could possibly use 3.7.4-slim-stretch instead, though, to lock down the patch version of Python as well.

Reply:
then, it lgtm